ISE: "No runnable rules found. Please ensure that at least one is enabled" when running smoke tests during the release procedure

Whenever `DiktatSaveSmokeTest` is [run during the release process](https://github.com/saveourtool/diktat/runs/7525484968?check_suite_focus=true#step:8:3933), the following command is executed during each test:

```bash
java -jar ktlint -R diktat.jar --disabled_rules=standard -F /tmp/FixPlugin--899173061/src/main/kotlin/DefaultPackageTest.kt
```

The command fails with

```stacktrace
java.util.concurrent.ExecutionException: java.lang.IllegalStateException: No runnable rules found. Please ensure that at least one is enabled.
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
	at com.pinterest.ktlint.internal.KtlintCommandLine.parallel(KtlintCommandLine.kt:578)
	at com.pinterest.ktlint.internal.KtlintCommandLine.parallel$default(KtlintCommandLine.kt:536)
	at com.pinterest.ktlint.internal.KtlintCommandLine.lintFiles(KtlintCommandLine.kt:283)
	at com.pinterest.ktlint.internal.KtlintCommandLine.run(KtlintCommandLine.kt:235)
	at com.pinterest.ktlint.Main.main(Main.kt:31)
Caused by: java.lang.IllegalStateException: No runnable rules found. Please ensure that at least one is enabled.
	at com.pinterest.ktlint.core.internal.RuleSorter.applyRunAfterRuleToRuleExecutionOrder(RuleSorter.kt:184)
	at com.pinterest.ktlint.core.internal.RuleSorter.getSortedRules(RuleSorter.kt:40)
	at com.pinterest.ktlint.core.internal.VisitorProvider.<init>(VisitorProvider.kt:35)
	at com.pinterest.ktlint.core.internal.VisitorProvider.<init>(VisitorProvider.kt:17)
	at com.pinterest.ktlint.core.KtLint.format(KtLint.kt:239)
	at com.pinterest.ktlint.internal.FileUtilsKt.formatFile(FileUtils.kt:202)
	at com.pinterest.ktlint.internal.KtlintCommandLine.process(KtlintCommandLine.kt:355)
	at com.pinterest.ktlint.internal.KtlintCommandLine.access$process(KtlintCommandLine.kt:48)
	at com.pinterest.ktlint.internal.KtlintCommandLine$lintFiles$3.invoke$lambda-1(KtlintCommandLine.kt:274)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

The output from the regular workflow:

```console
[INFO] Running org.cqfn.diktat.ruleset.smoke.DiktatSaveSmokeTest
[WARN ] 2022-07-26 17:05:29 Make sure JAVA_HOME (/opt/hostedtoolcache/Java_Zulu_jdk/11.0.16-8/x64) points to a Java 8 or Java 11 home. Java 17 is not yet supported.
[INFO ] 2022-07-26 17:05:29 The base directory for the smoke test is /home/runner/work/diktat/diktat/diktat-rules/src/test/resources/test/smoke.
[INFO ] 2022-07-26 17:05:29 Downloading https://github.com/saveourtool/save-cli/releases/download/v0.3.2/save-0.3.2-linuxX64.kexe to save-0.3.2-linuxX64.kexe...
[INFO ] 2022-07-26 17:05:30 6016712 byte(s) copied in 235.426 ms.
[INFO ] 2022-07-26 17:05:30 Downloading https://github.com/pinterest/ktlint/releases/download/0.46.1/ktlint to ktlint...
[INFO ] 2022-07-26 17:05:32 63363704 byte(s) copied in 732.33 ms.
[INFO] Tests run: 15, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 86.137 s - in org.cqfn.diktat.ruleset.smoke.DiktatSaveSmokeTest
```

The output from the release workflow:

```console
[INFO] Running org.cqfn.diktat.ruleset.smoke.DiktatSaveSmokeTest
[WARN ] 2022-07-26 17:17:36 Make sure JAVA_HOME (/opt/hostedtoolcache/Java_Zulu_jdk/11.0.16-8/x64) points to a Java 8 or Java 11 home. Java 17 is not yet supported.
[INFO ] 2022-07-26 17:17:36 The base directory for the smoke test is /home/runner/work/diktat/diktat/diktat-rules/src/test/resources/test/smoke.
[INFO ] 2022-07-26 17:17:36 Downloading https://github.com/saveourtool/save-cli/releases/download/v0.3.2/save-0.3.2-linuxX64.kexe to save-0.3.2-linuxX64.kexe...
[INFO ] 2022-07-26 17:17:37 6016712 byte(s) copied in 230.523 ms.
[INFO ] 2022-07-26 17:17:37 Downloading https://github.com/pinterest/ktlint/releases/download/0.46.1/ktlint to ktlint...
[INFO ] 2022-07-26 17:17:40 63363704 byte(s) copied in 2780.787 ms.
Error:  Tests run: 15, Failures: 15, Errors: 0, Skipped: 0, Time elapsed: 31.537 s <<< FAILURE! - in org.cqfn.diktat.ruleset.smoke.DiktatSaveSmokeTest
```

## Environment information
* diktat version: `1.2.2-SNAPSHOT` at 812406d8b4c7c506864d779d600e118b61f1b597

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -49,10 +49,15 @@ jobs:
               { "id": "gpg.passphrase", "passphrase": "${{ secrets.DIKTAT_GPG_PASS }}" },
               { "id": "kotlin-maven-plugin-tools", "username": "${{github.actor}}", "password": "${{ secrets.GITHUB_TOKEN }}" }
             ]
-      - name: Deploy artifacts
-        run: mvn -B clean deploy -Prelease --projects '!diktat-ruleset'
-      - name: Build diktat.jar
-        run: mvn -B package --projects diktat-ruleset
+        # Runs tests, incl. smoke tests (using a fat JAR built from the
+        # "diktat-ruleset" module).
+      - name: Run tests
+        run: mvn -B clean install
+        # "diktat-ruleset" is a module that creates a fat JAR, which is,
+        # obviously, never released. Since we *do need* this fat JAR in order to
+        # run smoke tests, we *skip tests entirely* in this step.
+      - name: Deploy artifacts (skipping the clean, test, and verify phases)
+        run: mvn -B deploy -Prelease --projects "!diktat-ruleset" -DskipTests=true
       - name: Create Github Release
         id: create_release
         uses: actions/create-release@v1

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -50,7 +50,8 @@ jobs:
               { "id": "kotlin-maven-plugin-tools", "username": "${{github.actor}}", "password": "${{ secrets.GITHUB_TOKEN }}" }
             ]
         # Runs tests, incl. smoke tests (using a fat JAR built from the
-        # "diktat-ruleset" module).
+        # "diktat-ruleset" module). The fat JAR is also used later during the
+        # "upload-release-asset" step (see below).
       - name: Run tests
         run: mvn -B clean install
         # "diktat-ruleset" is a module that creates a fat JAR, which is,

